2021-11-21 21:55:14,443 INFO    MainThread:76085 [wandb_setup.py:_flush():71] setting env: {'project': '1018_lw'}
2021-11-21 21:55:14,443 INFO    MainThread:76085 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-21 21:55:14,443 INFO    MainThread:76085 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211121_215514-w9j5zxol/logs/debug.log
2021-11-21 21:55:14,443 INFO    MainThread:76085 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211121_215514-w9j5zxol/logs/debug-internal.log
2021-11-21 21:55:14,444 INFO    MainThread:76085 [wandb_init.py:init():404] calling init triggers
2021-11-21 21:55:14,444 INFO    MainThread:76085 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-21 21:55:14,444 INFO    MainThread:76085 [wandb_init.py:init():449] starting backend
2021-11-21 21:55:14,444 INFO    MainThread:76085 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-21 21:55:14,464 INFO    MainThread:76085 [backend.py:ensure_launched():199] starting backend process...
2021-11-21 21:55:14,476 INFO    MainThread:76085 [backend.py:ensure_launched():204] started backend process with pid: 76146
2021-11-21 21:55:14,478 INFO    MainThread:76085 [wandb_init.py:init():458] backend started and connected
2021-11-21 21:55:14,480 INFO    MainThread:76085 [wandb_init.py:init():519] updated telemetry
2021-11-21 21:55:14,496 INFO    MainThread:76085 [wandb_init.py:init():549] communicating current version
2021-11-21 21:55:16,438 INFO    MainThread:76085 [wandb_init.py:init():554] got version response 
2021-11-21 21:55:16,439 INFO    MainThread:76085 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-21 21:55:16,968 INFO    MainThread:76085 [wandb_init.py:init():592] starting run threads in backend
2021-11-21 21:55:17,802 INFO    MainThread:76085 [wandb_run.py:_console_start():1816] atexit reg
2021-11-21 21:55:17,803 INFO    MainThread:76085 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-21 21:55:17,803 INFO    MainThread:76085 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-21 21:55:17,806 INFO    MainThread:76085 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-21 21:55:17,806 INFO    MainThread:76085 [wandb_init.py:init():619] run started, returning control to user process
2021-11-21 21:55:17,816 INFO    MainThread:76085 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'first_test_default_name', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 32, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': False, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00009_default_name', 'commit': 'not_set'}
2021-11-21 21:55:22,706 INFO    MainThread:76085 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2021-11-21 21:55:22,707 INFO    MainThread:76085 [wandb_run.py:_restore():1758] restore
